### DISCUSSION TITLE
I can't find Vitals in the latest version of Kong EE (v3.5.x)

### DISCUSSION THREAD

**Discussion by mkopeyka** [Kong/kong#12435] [Help] [answered]
Hello,

I have migrated from Kong EE 2.8.x to Kong EE 3.5.x and I can't find [Vitals](https://docs.konghq.com/gateway/2.8.x/vitals/) anymore.
Is it removed from latest version of kong?

I have set `KONG_VITALS="on"` but it didn't help

What replacement should I use to see this data?
![image](https://github.com/Kong/kong/assets/29103373/82bbf66e-0211-45f9-9421-3040a804ae37)

**Comment by guanlan** [comment, marked as answer]
As of the 3.5.0.0 release, the product component known as Vitals is deprecated and is no longer included in Kong Gateway Enterprise. Kong Konnect users can take advantage of our [API Analytics](https://docs.konghq.com/konnect/analytics/) offering, which provides a superset of Vitals functionality.
